packages/core/src/telemetry/types.ts

```diff
@@ -249,6 +249,7 @@ export class FlashFallbackEvent {
 export enum LoopType {
   CONSECUTIVE_IDENTICAL_TOOL_CALLS = 'consecutive_identical_tool_calls',
   CHANTING_IDENTICAL_SENTENCES = 'chanting_identical_sentences',
+  LLM_DETECTED_LOOP = 'llm_detected_loop',
```

Comment:
Is this fixing any particular looping issue? If this is a general cognitive loop then we should tackle this a different IMO